poo/atv_campus_ufc/CampusLista.py

```
@@ -0,0 +1,129 @@
+    def cadastrar_campus(self, nome_campus: str):
+        
+        nome_campus = nome_campus.lower()
+        if nome_campus in self.campus:
+            print(f"o campus '{nome_campus}' ja esta cadastrado\n")
+        self.campus[nome_campus] = []
+        print(f"o respectivo {nome_campus} foi adicionado \n")
+        return print(self.campus)
+    
+    def cadastrar_curso_campus(self, nome_campus: str, curso_novo: str):
+        nome_campus = nome_campus.lower()
+        if not nome_campus.lower() in self.campus:
```

Ruff (0.14.4)

Test for membership should be `not in`

Convert to `not in`

(E713)